snooty.toml

```
@@ -5,7 +5,7 @@ toc_landing_pages = ["/fundamentals/authentication", "/fundamentals", "/fundamen
 sharedinclude_root = "https://raw.githubusercontent.com/10gen/docs-shared/main/"
 
 [constants]
-version = 4.2
+version = 4.3
```

Comment (possible Issue): I believe Nathan is hoping that constant are written as strings going forward.

```suggestion
version = "4.3"
```